sympy/physics/tests/test_units.py

```diff
@@ -20,6 +20,8 @@ def test_units():
     t = Symbol('t')
     assert integrate(t*m/s, (t, 1*s, 5*s)) == 12*m*s
     assert (t * m/s).integrate((t, 1*s, 5*s)) == 12*m*s
+    
+    assert (year / week).evalf(15) == Float(52.17745571428571)
```

Comment:
Sorry for chiming in about this so late, and for confusing you by telling you the exact opposite of what other authorities have said, but I doubt that this test serves any useful purpose:
It does not allow easy manual verification that the test is correct. (In fact the test is harder to verify than the code that it tests, and that is the opposite of what a unit test should do. In fact we just saw that effect because we stumbled over ```year``` not being an integral multiple of ```day```.)
It does not fix any regression that ever happened.
It does not exercize any particular code path that is assumed to be prone to bugs.

I'd simply leave that test out.